instrument_observation_params.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Alternatively, this could live in a JSONB column in the instruments table.

Reply:
Perhaps check to see if this file exists or not (or is empty). In which case, this part of the Source page is not rendered.

Reply:
Done (see the handler method)